doc/user/INTERACTIONS.md

```diff
@@ -56,6 +56,7 @@ Note that some hotkeys can be available or not depending on the file being loade
 Camera Hotkeys:
 * <kbd>1</kbd>: front view camera.
 * <kbd>3</kbd>: right view camera.
+* <kbd>5</kbd>: toggle orthographic projection.
```

Comment:
@Meakk do you think we should have a command line option for this or only interaction is fine ?

Reply:
can be done in another PR though